adage/analyze/api.py

```diff
@@ -275,12 +275,37 @@ class Meta:
         queryset = Node.objects.all()
         resource_name = 'node'
         allowed_methods = ['get']
+        multiple_allowed_methods = ['post']
         filtering = {
             'name': ('exact', 'in', ),
             'heavy_genes': ('exact', ),  # New filter, see apply_filters().
             'mlmodel': ('exact', ),
         }
 
+    def prepend_urls(self):
+        return [
+            url((r'^(?P<resource_name>%s)/'
+                 r'post_multiple%s$') %
+                (self._meta.resource_name, trailing_slash()),
+                self.wrap_view('dispatch_multiple'),
+                name='api_post_multiple'),
+        ]
+
+    def dispatch_multiple(self, request, **kwargs):
+        return self.dispatch('multiple', request, **kwargs)
+
+    def post_multiple(self, request, **kwargs):
```

Comment:
So tastypie will automatically call this method when someone posts to the ``.../post_multiple`` url? Does it get called somehow behind the scenes when ``dispatch_multiple()`` gets called?